python/python/bystro/supervised_ppca/gf_dropout_pt.py

```diff
@@ -138,10 +140,10 @@ def fit(self, X, y, task="classification", progress_bar=True, seed=2021):
 
         Parameters
         ----------
-        X : np.array-like,(n_samples,n_covariates)
+        X : NDArray,(n_samples,n_covariates)
```

Comment:
`fit(self, X: NDArray, y: NDArray, task: str = "classification", progress_bar: bool = True, seed: int = 2021):` ; it may be that the task, progress_bar, seed are not needed because the type system infers them from the default values, if so, don't add those